pkg/kube/container.go

```diff
@@ -254,6 +263,7 @@ func (c *Container) Remove() error {
 	if err := c.cleanupFiles(false); err != nil {
 		return fmt.Errorf("could not cleanup container: %v", err)
 	}
+	c.imgInfo.Return(c.ID())
```

Comment:
If there is any errors within `Remove` call before this call, that means resources are never released, it doesn't apply only to this function, but if an error occur before `c.cleanupFiles` by example, `c.cleanupFiles` would never call. Since this call is idempotent for k8s, it may be safer to ignore errors here and always execute remove operation, thoughts ?

Reply:
Agree, will think on how to redo it in a nice way